heron/tmaster/src/cpp/manager/stateful-restorer.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __TMASTER_STATEFUL_RESTORER_H_
+#define __TMASTER_STATEFUL_RESTORER_H_
+
+#include <set>
+#include <string>
+#include "manager/tmaster.h"
+
+namespace heron {
+namespace tmaster {
+
+class StatefulRestorer {
+ public:
+  StatefulRestorer();
+  virtual ~StatefulRestorer();
+  // Start a new 2PC with this checkpoint_id
```

Comment:
Is 2PC 2 Phase Commit? If so can we expand that here and elsewhere in this patch. That abbreviation is not obvious to the code reader.